### PR TITLE
vlcstreamer: add desc

### PR DESCRIPTION
**Pull request by alexreg**
This seems to be the "official" description on the website.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

File: Casks/vlcstreamer.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
   url "https://hobbyistsoftware.com/Downloads/VLCStreamer/Versions/VLCStreamer_#{version}.dmg"
   name "VLC Streamer"
+  desc "Streaming by the power of VLC"
```
Comment (neutric) [inline Casks/vlcstreamer.rb:7]: Not familiar with the app, but from cursory glance it does not have functionality by itself but enables the Android/iOS app "VLC Streamer". I'd thus suggest `desc "VLC Streamer companion app"`.

Comment (alexreg) [reply Casks/vlcstreamer.rb]: Your right, the official description is a bit vague and more of a slogan. I've updated it to something more descriptive. (This app actually does the streaming to the mobile devices.)

Comment (neutric) [reply Casks/vlcstreamer.rb]: I was imprecise, sorry. What I meant: does this "helper app", as the homepage calls it, provide functionality if you do not use the VLC Streamer mobile app?

Comment (alexreg) [reply Casks/vlcstreamer.rb]: Not as far as I'm aware.

Comment (neutric) [reply Casks/vlcstreamer.rb]: Then I'd personally go with describing it a companion app as your desc sounds like it exposes a stream url that one can access with a media player of ones choosing. Feel free to leave it as is, though. Maintainers have the last word anyway. :)

Comment (alexreg) [reply Casks/vlcstreamer.rb]: Yeah, I do see your point... will see what one of them says, I suppose. Obviously happy to change it if desired.

**Pull request merged**